### PR TITLE
add conflicts couch view

### PR DESCRIPTION
**Pull request by snopoke**
To get some visibility into how big of an issue doc conflicts are in our DB we need a view that shows us which docs are have conflicts.

### CODE REVIEW

File: corehq/couchapps/doc_conflicts/views/conflicts/map.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function(doc) {
```
Comment (dimagimon) [inline corehq/couchapps/doc_conflicts/views/conflicts/map.js]: Missing name in function declaration.

**Comment by czue** [comment]
@snopoke let's hold off on merge until all other infrastructure stuff is smooth

**Comment by snopoke** [comment]
@czue yea

**Comment by czue** [comment]
merging and preindexing

**Pull request merged**